lerna.json

```diff
@@ -3,5 +3,7 @@
   "packages": [
     "packages/@glimmer/*"
   ],
+  "npmClient": "yarn",
+  "npmClientArgs": ["--no-lockfile"],
```

Comment:
Why?

Reply:
I don’t remember; didn’t we pair on this PR? 🎃 I don’t remember if this is disabled because it caused problems or just to preserve existing Lerna/npm semantics. If we use workspaces then these would get hoisted to the root yarn.lock anyway.